Zarebin/nextgen-py-5/branches/url/nextgen-py-5/url/views.py

```diff
@@ -0,0 +1,43 @@
+from django.shortcuts import render
+from django.http import HttpResponse
+import random
+import string
+from .models import UrlData
+from .forms import Url
+from django.shortcuts import redirect, render
+
+
+
+
+def index(request):
+    return HttpResponse("Url Shortener")
+
+def urlShort(request):
+    if request.method == 'POST':
+        form = Url(request.POST)
+        if form.is_valid():
+
+            slug = ''.join(random.choice(string.ascii_letters) for x in range(10))

```

Comment:
how do you check whether the randomly generated slug is unique in database?

Reply:
I should run a query on database. 
I will create query soon.

Reply:
thank 's for review, 
I will create a query and check if the slug is unique or not.